<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFormWidget</class>
 <widget class="QWidget" name="SearchFormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSwitchView">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushExpander">
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CustomSearchFormWidget" name="uiCustomContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <zorder>uiSimpleContainer_2</zorder>
    </widget>
   </item>
   <item>
    <widget class="SearchFormContainer" name="uiSimpleContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SearchViewWidget" name="uiSearchView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="pushSave">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="uiStoredFilters"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushClear">
         <property name="text">
          <string>&amp;Clear</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushSearch">
         <property name="text">
          <string>&amp;Search</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchFormContainer</class>
   <extends>QWidget</extends>
   <header>Koo.Search.SearchFormWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomSearchFormWidget</class>
   <extends>QWidget</extends>
   <header>Koo.Search.CustomSearchFormWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchViewWidget</class>
   <extends>QWidget</extends>
   <header>Koo.Search.SearchViewWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
